src/framework/button/_button.scss

```diff
@@ -71,6 +71,12 @@ $ctaButtonColor: #35d0a0;
  * 1. Fill in the background with the top gradient color to fix an IE11 bug
  *    where the transparent background color was showing through at the top.
  */
+.button--alert {
+  background: #f43434;
+  color: #ffffff;
+  border-color: #d03535;
+}
```

Comment:
File could get cleaned up with e.g. order and not repeating `.button` but reusing parent.